tests/PHPStan/Analyser/LegacyNodeScopeResolverTest.php

```diff
@@ -9118,10 +9118,6 @@ public function dataUnionProperties(): array
 				'UnionProperties\Bar|UnionProperties\Foo',
 				'$something->doSomething',
 			],
-			[
-				'UnionProperties\Bar|UnionProperties\Foo',
-				'$something::$doSomething',
```

Comment:
This throw an error in PHP, a static access to a instance property is not valid.